docs/pages/Cardinal/API-Reference/World.mdx

```diff
@@ -183,31 +185,41 @@ func main() {
 
 
 ```go
-func (w *World) RegisterComponents(components ...AnyComponentType) error
+func RegisterComponent[T metadata.Component](world *World) error
 ```
 #### Example
 
 ```go
 package main
 
 import (
-"log"
-"github.com/my-username/my-world-engine-project/component"
-"pkg.world.dev/world-engine/cardinal/ecs"
+    "log"
+    "github.com/my-username/my-world-engine-project/component"
+    "pkg.world.dev/world-engine/cardinal"
 )
 
 func main() {
     // ... world setup ...
 
-	err := ecs.RegisterComponent[LocationComponet](world)
+	err := cardinal.RegisterComponent[LocationComponet](world)
 	if err != nil {
 		log.Fatal(err)
 	}
-	err = ecs.RegisterComponent[AttackPowerComponent](world)
+	err = cardinal.RegisterComponent[AttackPowerComponent](world)
 	if err != nil {
 		log.Fatal(err)
 	}
-	err = ecs.RegisterComponent[HealthComponent](world)
+	err = cardinal.RegisterComponent[HealthComponent](world)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Alternative RegisterComponent pattern with less error checking:
+	err = errors.Join(
+        cardinal.RegisterComponent[LocationComponet](world),
+        cardinal.RegisterComponent[AttackPowerComponent](world),
+        cardinal.RegisterComponent[HealthComponent](world),
+    )
```

Comment:
formatting looks off here

Reply:
There were spaces instead of tabs. I inserted tabs in the rest of the go snippets in this file.